src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        colorSupplier.getRandomColor(ball);
```

Comment:
The `getRandomColor` method in `ColorSupplier` modifies the `Ball` object directly. It might be better to have `getRandomColor` return a color, which can then be set on the `Ball` object here. This would improve the separation of concerns and make the code more intuitive.